src/CosmosDBStudio.ViewModel/EditorTabs/QuerySheetViewModel.cs

```diff
@@ -48,7 +48,7 @@ public class QuerySheetViewModel : TabViewModelBase, ISaveable
             _title = string.IsNullOrEmpty(path)
                 ? "Untitled " + (++UntitledCounter)
                 : Path.GetFileNameWithoutExtension(path);
-            _text = querySheet.Text;
+            _text = querySheet.Text == string.Empty ? DefaultQuery : querySheet.Text;
```

Comment:
Maybe use `string.IsNullOrEmpty` here? In case `querySheet.Text` is null...

Reply:
Implemented 👍